packages/@o3r/localization/package.json

```diff
@@ -32,6 +32,7 @@
     "@o3r/logger": "workspace:^",
     "@o3r/schematics": "workspace:^",
     "@schematics/angular": "~15.2.0",
+    "chokidar": "^3.5.2",
```

Comment:
Why is it added? It is a dependency used by an exported function?
If it is only required by a builder, it may be `optional` as well